examples/python/custom_tools/latency_tracker_tool.py

```
@@ -0,0 +1,210 @@
+                        'average': sum(timings) / len(timings),
+                        'min': min(timings),
+                        'max': max(timings),
+                        'latest': timings[-1]
+                    }
+            return metrics
+    
+    def get_summary(self) -> Dict[str, Any]:
+        """Get summary of all tracked requests."""
+        with self._lock:
+            summary = {}
+            for request_id, phases in self._data.items():
```

Ruff (0.11.9)

Loop control variable `phases` not used within loop body

Rename unused `phases` to `_phases`

(B007)